app/src/main/java/com/framgia/forder/screen/updateProduct/UpdateProductViewModel.java

```diff
@@ -23,10 +23,12 @@
 import java.io.FileNotFoundException;
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Locale;
```

Comment:
ERROR: Unused import: java.util.Locale. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports